dotfiles/xmonad/xmonad.hs

```diff
@@ -159,7 +159,7 @@ scratchpads = [ NS "htop" "urxvt -name htop -e htop" (resource =? "htop") defaul
 -- Spawn pipes and menus on boot, set default settings
 --------------------------------------------------------------------------------------------------------------------
 myXmonadBar :: String
-myXmonadBar = "lemonbar -f \"-benis-uushi-medium-r-normal--11-90-75-75-p-58-iso10646-1\" -B \"black\" | bash"
+myXmonadBar = "~/essentials/scripts/lemonbar/header_bar.sh | lemonbar -f \"-benis-uushi-medium-r-normal--11-90-75-75-p-58-iso10646-1\" -B \"black\" | bash"
```

Comment:
Now, instead of having two `lemonbars`, one for xmonad input and one for the system status icons, now we have one and we unify their content via this header_bar script. No more resolution issues, just hit `super + q` to reload xmonad and your bar will resize and render.